experiment_prototype/scan_classes/averaging_periods.py

```diff
@@ -93,21 +93,21 @@ def __init__(self, ave_keys, ave_slice_dict, ave_interface, transmit_metadata,
         if self.intt is not None:  # intt has priority over intn
             for slice_id in self.slice_ids:
                 if self.slice_dict[slice_id]['intt'] != self.intt:
-                    errmsg = """Slice {} and {} are INTEGRATION or PULSE interfaced and do not have the
-                        same Averaging Period duration intt""".format(self.slice_ids[0], slice_id)
+                    errmsg = "Slices {} and {} are INTEGRATION or PULSE interfaced and do not have the" \
+                             " same Averaging Period duration intt".format(self.slice_ids[0], slice_id)
                     raise ExperimentException(errmsg)
         elif self.intn is not None:
             for slice_id in self.slice_ids:
                 if self.slice_dict[slice_id]['intn'] != self.intn:
-                    errmsg = """Slice {} and {} are INTEGRATION or PULSE  interfaced and do not have the
-                        same NAVE goal intn""".format(self.slice_ids[0], slice_id)
+                    errmsg = "Slices {} and {} are INTEGRATION or PULSE interfaced and do not have the" \
+                             " same NAVE goal intn".format(self.slice_ids[0], slice_id)
                     raise ExperimentException(errmsg)
 
-        for slice_id in self.slice_ids:
+        for slice_id in self.slice_ids:  # TODO: This test seems to be superseded by the test for scan boundary being the same between slices
```

Comment:
Not sure about this. I think someone could easily make the scanbound the same but accidentally forget a beam inside their beam_order. Also, beam_order and scan_bound do not necessarily have to be the same length (interleavesound for instance).